spec/controllers/users_controller_spec.rb

```diff
@@ -30,6 +30,20 @@
     end
   end
 
+  describe "#me" do
+    let(:user) { valid_user }
+    context "when it is a valid request", valid_request: true do
+      it "gets single user information" do
+        get :me, id: user.id, format: :json
```

Comment:
Could you try it again? otherwise, it's pointless/hackish going to `get users/1/me`, to fetch the current user, i know it allows you to pass an arbitrary id, but the point is if you're already able to determine the id, why do we need the `/me` route! it's hackish